src/components/Card/index.js

```diff
@@ -14,15 +14,7 @@ const HeaderTitle = props => (
   </Text>
 );
 
-const HeaderTabs = props => <Tabs {...props}>{props.children}</Tabs>;
-
-HeaderTabs.propTypes = {
-  top: PropTypes.bool
-};
-
-HeaderTabs.defaultProps = {
-  top: false
-};
+const HeaderTabs = props => <Tabs>{props.children}</Tabs>;
```

Comment:
you do still need to pass props through here... {...props}

Reply:
Added a new commit to fix :+1: